### DISCUSSION TITLE
love this project

### DISCUSSION THREAD

**Discussion by jp8311** [complexlogic/big-launcher#2] [General]
Just some encouragement. Really excited by this!
I really feel like there is a void for a program that does this well, and feels natural to use. 
I guess this has a lot to do with not really having a htpc-os option available that's remotely modern. I love your comittmant to at least providing a "skin" to emulate the experience. Thanks for your hard work!

**Comment by jp8311** [comment]
Following up with an immediate ADHD driven shower thought: what Linux flavour are you using to have a good htpc experience? I did a fair bit of reading but none really seem to currently support or prioritise support for htpc features ie hdr / tonemapping / etc. Outside of coreleec and the Kodi likes. Which is horribly dated in my view. 
I'd love a Linux htpc flavour that really just exists to be a launcher for jellyfin/steam with good media playing feature support!

**Reply by complexlogic** [reply]
Glad to hear you are interested in the project.

Unfortunately, there's no support for HDR in Linux yet, although it's being worked on. If that's a must-have feature for you, Windows is your only option currently.

I use Arch Linux in my setup. My two recommended distros for a Linux HTPC are Arch and Debian. You may be interested in the [Linux Setup Guide](https://github.com/complexlogic/flex-launcher/blob/master/docs/setup_linux.md) documentation that I wrote for Flex Launcher. Specifically, the "Kiosk Mode Setup" section, that's exactly how I have my HTPC set up. It just boots right up into Flex Launcher without having a desktop, so it's similar to a streaming stick feel.

**Comment by YeapGuy** [comment]
I fully agree! Very excited for this :).
I noticed in the readme that the project could use a logo. I suppose I could attempt one. Do you have any sort of idea of how it should look like, or even colors, style..?

**Reply by complexlogic** [reply]
@YeapGuy Thanks for your interest in the project and the offer to contribute. I do indeed need help with the logo. However, the program name still hasn't been determined yet, and we would need to need a name defined so we can have a proper idea of what a logo for it would look like.

I'm thinking a name with a suffix of -TV, inspired by Amazon's "FireTV" product. So a cool-sounding noun or adjective, plus "TV". If you can think of anything that sounds good, let me know.

Another thing I need help with is designing the default background. I've experimented with a few different backgrounds (see below), but I haven't found anything that's quite satisfying yet. The background should be colorful - but not too colorful that it makes poor contrast with the images on top. Something like a color gradient or a basic pattern would be good I think. The background should be in SVG format so we can render it at any resolution.

Any contribution you are able to make would be greatly appreciated.

![screen41](https://user-images.githubusercontent.com/95071366/211718641-133099cf-f70c-4aa5-89d6-28b63659e915.png)

**Comment by Okayaceyok** [comment]
Hello! Im loving this project. Actually using it haha. Is it still in development?

**Comment by Fonzi03** [comment]
Checking in to see if this is still in development? 

Has a name suggestion based off your notion of -TV suffix. 

PCTV
Or
TV+


I also would like to request if it's not already built in pass through of Mic to the app for keyboard searching. Better yet if it can work similar to Google TV where it can decide which app has it streaming and open said app straight to the selected media. I know this is a big all but figured I'd throw it out there.
